NEWS.md

```diff
@@ -35,6 +36,81 @@ The format is based on [Keep a Changelog]
+of regression warnings when using wlroots 0.19. We would like to fix these
+before releasing:
+
+- Press-drag-release menu actions broken [#2787]
+- Blurry Gtk surfaces after output on/off with scale != 1.0 [#2769]
+
+### Added
+
+- Bump `wl_compositor` version from 5 to 6 @tokyo4j [#2812]
+- Support tablet tool mouse buttons @jp7677 [#2778]
+- Add libinput config options:
+  - `<threeFingerDrag>` @m4rch3n1ng [#2795]
+  - `<dragLock>sticky</dragLock>` @tokyo4j [#2803]
+- Support xdg-toplevel-icon protocol @tokyo4j [#2755]
+- Add `{left,right}-occupied` options to `GoToDesktop` @DreamMaoMao
+  [#2790]
+- Add config option `<theme><dropShadowsOnTiled>` @diredocks [#2789]
+- Add missing tracking of configure serials for xdg-shell surface to fix
+  issue with mpv @jlindgren90 [#2774] [#2788]
+- Add scroll method libinput option @Consolatis [#2767]
+
+```xml
+    <libinput>
+      <device>
+        <scrollMethod>none|twofinger|edge</scrollMethod>
+      </device>
+    </libinput>
+```
+
+- Support `drm-syncobj` protocol @zeusgoose [#2737]
+- Support `ext-image-copy-capture` protocol @any1 [#2740]
+- Support both axis for XWayland client side maximize requests.
+  @Consolatis [#2728]
+- Add scroll emulation for cursor motion and associated actions @jp7677
+  [#2678]:
+   - `EnableScrollWheelEmulation`
+   - `DisableScrollWheelEmulation`
+   - `ToggleScrollWheelEmulation`
+
+### Fixed
+
+- Fix crash relating to the window `set_icons` signals @Consolatis [#2799]
+- Arrange layers on layer-shell surface `destroy` rather than `unmap` to
+  fix issues with `wshowkeys` and `kitten` @johanmalm [#1153] [#1154]
+- Window switcher fixes: @tokyo4j [#2770]
+  - Always show title with `<field content="title">`. Before this patch,
+    titles were not shown if identical to identifiers.
+  - Always show output name with `<field content="output">`. Before this
+    patch, output names were not shown if there was only one output.
+- Send fractional scale to layer-shell surfaces before map. @Consolatis [#2768]
+- Only configure initialized layer-shell surfaces to fix bug with
+  `kitten quick-access-terminal` @alex-huff [#2736] [#2745] 
+- Improve focus semantics for XWayland windows using the Globally Active
+  input model to fix issues with Zoom and WeChat @jlindgren90 [#1142] [#2811]
+- Provide better support for XWayland client keyboard focus grabs by using
+  the new `grab_focus` signal. @jlindgren90 [#1142]
+- Guard against negative sizes in window-switching and menu graphical
+  artefacts. @tokyo4j [#2727]
+- Do not broadcast keyboard modifiers from virtual keyboards to fix issue
+  with per-window layout settings. @orfeasxyz [#2723] [#2724]
+- Gracefully exit when no fonts are installed @tokyo4j [#2713]
+- config: validate total osd field width to ensure it does not exceed
+  100%.  @tokyo4j [#2710]
+
+### Changed
+
+- "Sticky" mode for drag-locking is enabled by default as recommended by
+  libinput [#2803]
```

Comment:
Maybe add something like

The timeout based behavior can be restored via
```xml
    <libinput>
      <device>
        <dragLock>yes</dragLock>
      </device>
    </libinput>
```
To disable the feature all together `no` can be used as value.

CC @jlindgren90

Reply:
Done.